pkgs/development/tools/misc/loccount/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, lib, buildGoPackage, fetchgit }:
+buildGoPackage rec {
+  name = "loccount-${version}";
+  version = "20170213-${stdenv.lib.strings.substring 0 7 rev}";
+  rev = "7c45bc55394cd065f6a966110f5f8c3478998e95"; # 1.0 release
+
+  goPackagePath = "gitlab.com/esr/loccount";
+  excludedPackages = "tests";
+
+  src = fetchgit {
```

Comment:
where are these fetch commands defined? I tried to find this one but could not...